### ISSUE TITLE
Fix "from" in meta template expansion to use a non-nodemailer var name

### ISSUE THREAD

**Issue by nexdrew** [npm/email-templates#18]
Currently the meta templates contain JSON similar to:

```json
{
  "subject": "Your npm On-Site trial license key and instructions",
  "to": "\"{{name}}\" <{{email}}>",
  "from": "\"npm, Inc.\" <{{from}}>"
}
```

But doing `"from": "\"npm, Inc.\" <{{from}}>"` is problematic because the data given to expand `{{from}}` _will overwrite_ the `"from"` meta field entirely instead of expanding it. This is self-defeating, rendering the `"from"` meta field useless.

The result of this problem is that all emails will only use the address `{{from}}` (e.g. `website@npmjs.com`) instead of the desired format `"npm, Inc." <{{from}}>` (e.g. `npm, Inc. <website@npmjs.com>`), as evidenced by emails sent from npm/newww.

To fix, we can simply change the template variable `{{from}}` to something unique (i.e. not expected by nodemailer) like `{{fromEmail}}`. This change can be made without affecting existing functionality of npm/newww since the [data it gives](https://github.com/npm/newww/blob/v4.0.1/services/email/methods/send.js#L10) overwrites the `"from"` meta field anyway. The meta templates in this module should change first, then the field name provided by npm/newww can change to complete the fix.

**Comment by nexdrew** [comment]
This problem will be fixed with npm/mustache-mailer#5.

**Comment by nexdrew** [comment]
Fixed with npm/newww#1802. See details (links to all PRs) there.
